### DISCUSSION TITLE
Filter queries with repositories that contains files

### DISCUSSION THREAD

**Discussion by themisir** [community/community#9981] [Other Feature Feedback, Questions, & Ideas]
I recently wanted to see how a public OSS library I'm working on being consumed by other libraries. And thought I might use Code Search to check for specific method calls. And it actually worked out as expected.

But I still needed a way to filter search results to only contain library repositories rather than library and app repos, I can do that if I could filter search results by repos that contains specific file for example package manifest or CHANGELOG.md that's usually pushed along with library repos rather than apps.

In short filters like that would be awesome:

```
# Filter repositories that contains lib.rs
contains:/src/lib.rs

# Repositories that has a file path matching "/**/manifest.yaml" glob and contents containing "type: library" string
contains:"type: library" /**/manifest.yaml
```
